src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,10 @@
+import { TodoInfo } from '../TodoInfo';
+
 // Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+export const TodoList = ({ todos }) => (
```

Comment:
The component lacks error handling. It is important to check if the todos prop is not null or undefined before mapping over it